osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SystemEventServiceTest.java

```diff
@@ -141,8 +140,9 @@ void verifyMaxValueReachedEvent() {
     final MessageMetadata metadata = messageMetadataCaptor.getValue();
     assertThat(metadata.getDeviceIdentification()).isEqualTo(device.getDeviceIdentification());
     assertThat(metadata.getCorrelationUid()).isEqualTo("corr-id");
-    assertThat(metadata.getOrganisationIdentification()).isEqualTo("no-organisation");
-    assertThat(metadata.getIpAddress()).isNull();
+    assertThat(metadata.getOrganisationIdentification())
+        .isEqualTo(metadata.getOrganisationIdentification());
+    assertThat(metadata.getIpAddress()).isEqualTo(metadata.getIpAddress());
```

Comment:
This looks like checking metadata properties are equal to themselves.
Shouldn't `metadata` for the expected values be replaced by `messageMetadata`?

Reply:
But of course, my bad! Changed